CRM/Import/Parser.php

```diff
@@ -1288,7 +1291,7 @@ protected function getInvalidValues($value, string $key = '', string $prefixStri
     if ($value === 'invalid_import_value') {
       if (!is_numeric($key)) {
         $metadata = $this->getFieldMetadata($key);
-        $errors[] = $prefixString . ($metadata['html']['label'] ?? $metadata['title']);
+        $errors[] = $prefixString . ($metadata['label'] ?? $metadata['html']['label'] ?? $metadata['title']);
```

Comment:
Does `$metadata['html']['label']` contain HTML or text? Does it need a `strip_tags` or something similar?

Reply:
@Sjord no - this is just from metadata - eg. Membership Type for the field membership_type_id